src/expression.js

```diff
@@ -73,6 +74,10 @@ var getKeyComputeData = function (key, scope, readOptions) {
 			return computeOrFunction();
 		}
 
+		if (computeOrFunction.computeData != null && computeOrFunction.hasOwnProperty("value")) {
```

Comment:
`Bracket.protoype.value` calls `lookupValue` first when the bracket prop is a `Lookup` -- the result of `lookupValue` always has a `computeData` prop -- then sends the result as the `keyOrCompute` param to `lookupValueInResult`.  When the compute created in `lookupValueInResult` fires its gettersetter, it calls `getValueOfComputeOrFunction(keyOrCompute)` and pulls that `lookupValue` created function out of the outer function's context.